_episodes/11-practice-teaching.md

```diff
@@ -164,10 +112,11 @@ Here is a list of different
 ways that you can set the stage for receiving or providing feedback
 in a way that facilitates growth.
 
-*   Initiate feedback as the instructor. It is better to ask for feedback than to receive it unwillingly.
+*   **Initiate feedback.** Most people will not offer it freely, and those who do are not the only voices you should listen to. Also, it is easier to 
```

Comment:
This comment is about the previous section:
1. Can we include a little image of the 2x2 feedback rubric or at least pull that info out with bold text or a highlight box? I think it is easy to miss it otherwise.
2. As always I will put in a plug for a language change to "constructive feedback" versus "negative feedback"